### ISSUE TITLE
Fix wazuh timestamps

### ISSUE THREAD

**Issue by davidjiglesias** [wazuh/wazuh#3784]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
|3.9.3|Framework|All|All|All|

Hello team, we have discovered a problem with how the framework is processing timestamps. Wazuh framework is actually using `datetime.fromtimestamp` which transforms the timestamp into a local date. Such transformation is creating discrepancies with Wazuh APP visualizer since they assume the date provided by the API is in UTC format. 

We should use `datetime.utcfromtimestamp` to ensure the timestamp maintains its integrity all the way through. We will thoroughly review every other possible application for this issue at API and framework level.

Best regards,

David J. Iglesias

**Comment by davidjiglesias** [comment]
Status update:

**Comment by vikman90** [comment]
@davidjiglesias Thank you for investigating this issue.

Let me add the label `core/db` and ping @wazuh/core so we will track this issue.

Cheers!

**Comment by SalvaMilenario** [comment]
Hi team,

I did changes in the code SQL that create table `pm_event` [(51a378c)](51a378c205cc83d3a82829a9009470819d29e637), `agent` [(e6487d2)](e6487d22c9abe438316f35b65c1ea6b8373fe86d) and insert the agent 000 in the table `agent` [(914c20e)](914c20e706e6fe60b0d4e0585adef42b376c7a5c) for that, the time is saved as integer instead of text. In addition, I modified the functions `wdb_insert_agent` and `get_agent_date_added`to save in database the time as integer. Those integers are in the timestamp of UTC. 

Regards.
